test/fail_compilation/invalid_lib.d

```diff
@@ -1,7 +1,7 @@
 /*
 REQUIRED_ARGS: -lib
-REQUIRED_ARGS(linux freebsd osx openbsd) fail_compilation/extra-files/fake.a
-REQUIRED_ARGS(windows) -m32 fail_compilation/extra-files/fake.lib
+REQUIRED_ARGS(linux freebsd osx openbsd): fail_compilation/extra-files/fake.a
+REQUIRED_ARGS(windows): -m32 fail_compilation/extra-files/fake.lib
```

Comment:
Where did the (...) syntax come from? This is not documented.

Reply:
Apparently missing from the README, will add in another PR.